docs/partials/install-nrf91-sdk.md

```diff
@@ -38,7 +37,7 @@ Tell `west` to automatically configure CMake:
 west zephyr-export
 ```
 
-Lastly, install the remaining dependencies:
+Install the remaining dependencies:
```

Comment:
This should not be part of `Remove west patch step, fixes #134` commit.

Reply:
I reverted this commit and separated the two changes. The `west patch` step is now removed in 9275e23

Reply:
Revert commits just pollute commit logs and make PRs potentially reach indefnite number of commits (one reverting the other). This makes review harder, especially for people like me that do review commit by commit (which IMO should be the case for everybody, as each commit should have a rationale why specific change is needed, so it is hard to review without looking at git commit message and potentially approve that code/doc changes are really doing what was intended by commit author).

Reply:
Okay, that makes sense. What is a better way to fix this situation?

Reply:
Remove a commit (instead of doing a revert) and doing a force push to this branch (force push is fine as long as people do not base their work on those branches - this is still a PR, so it should be fine). This is what Zephyr RTOS recommendation is, what open source projects did before Github was used (commit was removed before resending patches over to mainling list, e.g. in case of Linux) and what I also use in other projects. In total this will remove two commits from PR (original one and revert).

Though, I am not maintainer of this repo, so I don't want to introduce too much policy. This is more like my opinion here to keep repo maintainable and PR reviewable.

Reply:
Appreciate the insight, Mike just was able to successfully execute the force push

Reply:
Yes, thanks for the pointer!